Automattic/WP-Job-Manager
# Fix/issue 1912

Fixes #1912 

#### Changes proposed in this Pull Request:

* The paginate_links() function adds the url args (from the current url) to each pagination link. The actions (mark as filled, not filled, etc) in the job dashboard were triggered with the url (For example: url/?action=mark_as_filled). The consequences of this were that if you changed the page, the pagination link would carry the "action, job_id, and _wpnonce" parameters to the new page, causing the action to be triggered again; thus, getting the message that the job was already filled/not filled. 
* This PR removes the args from the url for the pagination links so that the actions are not triggered again when changing pages.

#### Testing instructions:

1. Make sure you have enough jobs on your jobs dashboard that they are paginated at least to a page 2.
2. On the jobs dashboard, mark a job as filled; you will get the correct "X has been filled" message.
3. Click on the "2" or the "next arrow" in the pagination at the bottom of the jobs list.
4. You should not see any messages


#### Proposed changelog entry for your changes:
* Remove args from pagination_link urls
